scripts/fetchLatestSubstrate.js

```diff
@@ -0,0 +1,116 @@
+// curl https://getsubstrate.io -sSf | bash
+
+const { existsSync, writeFileSync } = require('fs');
+const fetch = require('node-fetch');
+const path = require('path');
+const { promisify } = require('util');
+const semver = require('semver');
+
+const {
+    substrate: { version: versionRequirement }
+} = require('../packages/electron-app/package.json');
+
+const exec = promisify(require('child_process').exec);
+var spawn = require('child_process').spawn;
+
+const ENDPOINT = 'https://getsubstrate.io';
+
+const STATIC_DIRECTORY = path.join(
+    '..',
+    'packages',
+    'electron-app',
+    'static'
+);
+
+const foundPath = [
+    path.join(STATIC_DIRECTORY, 'substrate'),
+    path.join(STATIC_DIRECTORY, 'substrate.exe')
+].find(existsSync);
+
+if (foundPath) {
+    // Bundled Parity Substrate was found, we check if the version matches the minimum requirements
+    getBinaryVersion(foundPath)
+        .then(version => {
+            if (!version) {
+                console.log("Couldn't get bundled Parity Substrate version.");
+                return downloadSubstrate();
+            }
+
+            if (!semver.satisfies(version, versionRequirement)) {
+                console.log(
+                    'Bundled Parity Substrate %s is older than required version %s',
+                    version,
+                    versionRequirement
+                );
+                return downloadSubstrate();
+            } else {
+                console.log(
+                    'Bundled Parity Substrate %s matches required version %s',
+                    version,
+                    versionRequirement
+                );
+                // FIXME: change to runSubstrateLight once that's functioning
+                runSubstrateDev();
```

Comment:
This script should only fetch substrate. The running part should be done by electron